exec/java-exec/src/main/resources/rest/static/js/graph.js

```diff
@@ -93,41 +93,82 @@ $(window).on('load',(function () {
     }
 
     // parse the short physical plan into a dagreeD3 structure
-    function parseplan (planstring) {
+    function parseplan (planstring, implicitSrcMap) {
```

Comment:
That is because the caller function is `buildplangraph`, which is already returning the graph object. I tried making a global variable, but it didn't work.